test/__snapshots__/jsToAqua.test.ts.snap

```diff
@@ -0,0 +1,127 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Conversion from js to aqua Expect objects to match snapshot 1`] = `
+{
+  "type": "[]u64",
+  "typeDefs": undefined,
+  "value": "[
+        1,
+        2,
+        3
+]",
+}
+`;
```

Comment:
What are inputs, what are outputs? What is tested against what?

Reply:
I improved snapshots and now you can see inputs and outps clearly in the snapshot itself